src/main.rs

```diff
@@ -16,29 +16,56 @@ use tokio_core::reactor::Core;
 use name_gen::random_msg;
 
 fn main() {
-    let token = get_token(); 
-
-    let epic = random_msg();
-    let tweet = DraftTweet::new(&epic);
-    
-    let mut core = Core::new().unwrap();
-    let handle = core.handle();
-    core.run(tweet.send(&token, &handle)).unwrap();
+    let token = get_token();
+
+    if token.is_err() {
```

Comment:
Don't do the `is_err()` and `unwrap()` dance here, it's dangerous.
Instead, write this as

```rust
if let Ok(token) = token {
    // success case, token is now the contents of Ok
} else {
    // error case
}
```